### PR TITLE
Add 3.2.0 changelog

### PR DESCRIPTION
**Pull request by yesmeck**

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,33 @@ timeline: true
 
 ---
 
+# 3.2.0
+
+`2018-02-04`
+
+- 🌟 Add new `tabBarGutter` prop to Tab to allow setting gutter between tabs. [#8644](https://github.com/ant-design/ant-design/pull/8644) [@lixiaoyang1992](https://github.com/lixiaoyang1992)
+- 🌟 Add new `hasSider` prop to Layout to avoid render error when server rendering. [#8937](https://github.com/ant-design/ant-design/issues/8937)
+- 🌟 Add new `successPercent` to Progess to alow showing two phases of progess. [Demo](https://ant.design/components/progress/#components-progress-demo-segment)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: to Progess to alow showing two phases of progess -> to Progress to allow showing two phases

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+- 🌟 Add new `iconType` prop to Alert to allow settting type of icon. [#8811](https://github.com/ant-design/ant-design/pull/8811) [@minwe](https://github.com/ant-design/ant-design/pull/8811)
+- 🌟 Add `id` propm to DatePicker. [#8598](https://github.com/ant-design/ant-design/pull/8598) [@mgrdevport](https://github.com/mgrdevport)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: propm -> prop

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+- 🌟 Add new `forceRender` prop to Collapse to allow rendering hide panel. [#9192](https://github.com/ant-design/ant-design/pull/9192) [#Pyroboomka](https://github.com/ant-design/ant-design/pull/9192) [@paulcmason](https://github.com/react-component/collapse/pull/82)
+- RangePicker
+  - 🌟 Improve `ranges` prop to allow passig function to it. [#8281](https://github.com/ant-design/ant-design/issues/8281)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: passig -> passing

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+  - 🐞 Fix issue resulting in can not input start dated manually. [#6999](https://github.com/ant-design/ant-design/issues/6999)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: dated -> date

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+  - 🐞 Fix issue resulting date panel closed without animation when click on the preset range. [#6364](https://github.com/ant-design/ant-design/issues/6364)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: resulting date panel closed -> resulting in date panel being closed

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+  - 🐞 Fix issue resulting `onOk` is not trigged, when click on the preset range. [#7747](https://github.com/ant-design/ant-design/issues/7747)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: resulting `onOk` is not trigged -> resulting in `onOk` is not being triggered

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+- Select
+  - 🌟 Improve `onChange`、`onDeselect`, they will receive selected `Option` as second paramteter.
+  - 🐞 Fix issue resulting in `onSelect` is not trigged when using automatic tokenization. [#9094](https://github.com/ant-design/ant-design/issues/9094)
+  - 🐞 Fix the missing scrollbar in Chrome.
+- 🌟 Improve Table's `rowSelection[getCheckboxProps]` prop, now the all of returns of `getCheckboxProps` will pass to checkbox. [#9054](https://github.com/ant-design/ant-design/pull/9054) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9054)
+- 🐞 Fix issue resulting in can not change Calendar's `mode` from outside. [#9243](https://github.com/ant-design/ant-design/pull/9243) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9243)
+- 🐞 Fix issue resulting AutoComplete shows wrong border when showing validattion message.[9f2b490](https://github.com/ant-design/ant-design/commit/9f2b4905f09fca503da7a8bb5f2b8347bea663b7)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: resulting AutoComplete shows wrong border when showing validattion  -> resulting in AutoComplete showing wrong border when showing validation

File: CHANGELOG.en-US.md [hunk 8]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+- 🐞 Fix issue resulting in DatePicker shows wrong time in the control mode. [#8885](https://github.com/ant-design/ant-design/issues/8885)
+- 🐞 Fix issue resulting in TextArea shows wrong height in Edge. [#9108](https://github.com/ant-design/ant-design/pull/9108) [@cuyl](https://github.com/cuyl)
+- 🐞 Fix issue resulting in Tabs shows wrong tab style when using `type="card"` combine with `tabPosition="bottom"`. [#9165](https://github.com/ant-design/ant-design/pull/9165) [@ryanhoho](https://github.com/ryanhoho)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: same

File: CHANGELOG.en-US.md [hunk 9]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+- Select
+  - 🌟 Improve `onChange`、`onDeselect`, they will receive selected `Option` as second paramteter.
+  - 🐞 Fix issue resulting in `onSelect` is not trigged when using automatic tokenization. [#9094](https://github.com/ant-design/ant-design/issues/9094)
+  - 🐞 Fix the missing scrollbar in Chrome.
+- 🌟 Improve Table's `rowSelection[getCheckboxProps]` prop, now the all of returns of `getCheckboxProps` will pass to checkbox. [#9054](https://github.com/ant-design/ant-design/pull/9054) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9054)
+- 🐞 Fix issue resulting in can not change Calendar's `mode` from outside. [#9243](https://github.com/ant-design/ant-design/pull/9243) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9243)
+- 🐞 Fix issue resulting AutoComplete shows wrong border when showing validattion message.[9f2b490](https://github.com/ant-design/ant-design/commit/9f2b4905f09fca503da7a8bb5f2b8347bea663b7)
+- 🐞 Fix issue resulting in DatePicker shows wrong time in the control mode. [#8885](https://github.com/ant-design/ant-design/issues/8885)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: shows -> showing

File: CHANGELOG.en-US.md (hunk 8 above)
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: same

File: CHANGELOG.en-US.md [hunk 10]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+  - 🐞 Fix issue resulting in can not input start dated manually. [#6999](https://github.com/ant-design/ant-design/issues/6999)
+  - 🐞 Fix issue resulting date panel closed without animation when click on the preset range. [#6364](https://github.com/ant-design/ant-design/issues/6364)
+  - 🐞 Fix issue resulting `onOk` is not trigged, when click on the preset range. [#7747](https://github.com/ant-design/ant-design/issues/7747)
+- Select
+  - 🌟 Improve `onChange`、`onDeselect`, they will receive selected `Option` as second paramteter.
+  - 🐞 Fix issue resulting in `onSelect` is not trigged when using automatic tokenization. [#9094](https://github.com/ant-design/ant-design/issues/9094)
+  - 🐞 Fix the missing scrollbar in Chrome.
+- 🌟 Improve Table's `rowSelection[getCheckboxProps]` prop, now the all of returns of `getCheckboxProps` will pass to checkbox. [#9054](https://github.com/ant-design/ant-design/pull/9054) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9054)
+- 🐞 Fix issue resulting in can not change Calendar's `mode` from outside. [#9243](https://github.com/ant-design/ant-design/pull/9243) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9243)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: resulting in can not change Calendar's `mode` from outside -> resulting in Calendar's `mode` not being allowed to be changed from outside

File: CHANGELOG.en-US.md [hunk 11]
```diff
@@ -15,6 +15,33 @@ timeline: true
...
+- RangePicker
+  - 🌟 Improve `ranges` prop to allow passig function to it. [#8281](https://github.com/ant-design/ant-design/issues/8281)
+  - 🐞 Fix issue resulting in can not input start dated manually. [#6999](https://github.com/ant-design/ant-design/issues/6999)
+  - 🐞 Fix issue resulting date panel closed without animation when click on the preset range. [#6364](https://github.com/ant-design/ant-design/issues/6364)
+  - 🐞 Fix issue resulting `onOk` is not trigged, when click on the preset range. [#7747](https://github.com/ant-design/ant-design/issues/7747)
+- Select
+  - 🌟 Improve `onChange`、`onDeselect`, they will receive selected `Option` as second paramteter.
+  - 🐞 Fix issue resulting in `onSelect` is not trigged when using automatic tokenization. [#9094](https://github.com/ant-design/ant-design/issues/9094)
+  - 🐞 Fix the missing scrollbar in Chrome.
+- 🌟 Improve Table's `rowSelection[getCheckboxProps]` prop, now the all of returns of `getCheckboxProps` will pass to checkbox. [#9054](https://github.com/ant-design/ant-design/pull/9054) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9054)
```
Comment (abenhamdine) [inline CHANGELOG.en-US.md]: I'm not sure about this one : 

now the all of returns of `getCheckboxProps` will pass to checkbox -> now all the properties returned by `getCheckboxProps` will be passed to checkbox

**Review by abenhamdine — CHANGES_REQUESTED** [review]

File: CHANGELOG.en-US.md (hunk 3 above)
Comment (yesmeck) [reply CHANGELOG.en-US.md]: Thanks, I do need a grammar checker for my editor. 😄

**Pull request merged**